src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -478,22 +491,30 @@ def main():
     conf = None
 
     try:
+        # setup container.conf file for the callback server logs if needed
+        if loader_helper.is_config_modification_required():
+            if click.confirm(f"The config file at {loader_common_names.CONTAINERS_CONF_PATH}\n"
+                             f"needs to be modified to allow for container logging. Do so now?\n"):
+                loader_helper.setup_callback_server_logs()
+            else:
+                print("Permission denied and exiting ...")
+                return
+
         # start podman service
         proc = loader_helper.start_podman_service(uid)
 
-        # set up conf, start callback server, and upload assemblies to workspace
+        # set up conf for uploader, start callback server, and upload assemblies to workspace
         conf = Conf(
-            job_dir,
-            output_dir,
-            _process_input,
-            kb_base_url,
-            token_filepath,
-            workers,
+            job_dir=job_dir,
+            kb_base_url=kb_base_url,
+            token_filepath=token_filepath,
+            max_task=max_task,
+            workspace_downloader=False,
```

Comment:
Maybe we should make the default arg here False and remove this line, so that when the downloader stuff is removed from the CBS wrapper we don't need changes here as well

Reply:
Updated!

Reply:
:+1: